packages/serverpod/lib/src/database/adapters/postgres/database_connection.dart

```diff
@@ -542,12 +542,26 @@ class DatabaseConnection {
   }
 
   /// For most cases use the corresponding method in [Database] instead.
-  Future<R> transaction<R>(TransactionFunction<R> transactionFunction) {
+  Future<R> transaction<R>(
```

Comment:
Intentionally left as an unresolved conversation.

Don't merge this until 2.2 has been released.